org.eclipse.xtext.tests/src/org/eclipse/xtext/tests/AbstractXtextTests.java

```diff
@@ -390,6 +397,30 @@ protected String readFileIntoString(String filePath) throws IOException {
 		throw new IllegalStateException("May not happen, but helps to suppress false positives in eclipse' control flow analysis.");
 	}
 
+	protected Grammar load(URI uri) {
```

Comment:
As far as I can tell, AbstractXtextTests was meant to be used for various test languages in this test bundle. Having a method `load(URI)` that expects the resource to point to an Xtext grammar file, appears to be wrong.